manifests/istio-telemetry/grafana/dashboards/pilot-dashboard.json

```diff
@@ -1586,7 +1792,7 @@
     ]
   },
   "timezone": "browser",
-  "title": "Istio Pilot Dashboard",
+  "title": "Istio Control Plane Dashboard",
```

Comment:
If you rename this to Control Plane, do we need to update the file name to `control-plane-dashboard.json` also in the `dashboard_test.go` where has `pilot` part?